sql/analyzer/resolve_orderby.go

```diff
@@ -55,7 +58,7 @@ func pushdownSort(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope) (sql.
 				name := strings.ToLower(n.Name())
 				if stringContains(childAliases, name) {
 					colsFromChild = append(colsFromChild, n.Name())
-				} else if !stringContains(schemaCols, name) {
+				} else if !tableColsContains(schemaCols, tableColFromNameable(n)) {
```

Comment:
Seems obv correct, but make sure you add a test that breaks before and works now